man/roll.Rd

```diff
@@ -0,0 +1,70 @@
+\name{roll}
```

Comment:
html preview of latest manual in roll branch (in my gitlab repo)  on: https://jangorecki.gitlab.io/data.table/library/data.table/html/roll.html